packages/ember-simple-auth/tests/unit/internal-session-test.js

```
@@ -23,307 +22,343 @@ describe('InternalSession', () => {
         });
 
-        itHandlesAuthenticatorEvents(async() => {
+        itHandlesAuthenticatorEvents(async function() {
           await session.restore();
         });
       });
 
-      describe('when the authenticator rejects restoration', function() {
-        beforeEach(function() {
+      module('when the authenticator rejects restoration', function(hooks) {
+        hooks.beforeEach(function() {
           sinon.stub(authenticator, 'restore').returns(RSVP.reject());
         });
 
         itDoesNotRestore();
       });
     });
 
-    describe('when the restored data does not contain an authenticator factory', function() {
+    module('when the restored data does not contain an authenticator factory', function() {
       itDoesNotRestore();
     });
 
-    describe('when the store rejects restoration', function() {
-      beforeEach(function() {
+    module('when the store rejects restoration', function(hooks) {
+      hooks.beforeEach(function() {
         sinon.stub(store, 'restore').returns(RSVP.Promise.reject());
       });
 
-      it('is not authenticated', async function() {
-        await session.restore();
+      test('is not authenticated', async function(assert) {
+        try {
+          await session.restore();
+        } catch (e) {
+          // ignore
+        }
 
-        expect(session.get('isAuthenticated')).to.be.false;
+        assert.notOk(session.get('isAuthenticated'));
       });
     });
 
-    describe('when the store rejects persistance', function() {
-      beforeEach(function() {
+    module('when the store rejects persistance', function(hooks) {
+      hooks.beforeEach(function() {
         sinon.stub(store, 'persist').returns(RSVP.reject());
       });
 
-      it('is not authenticated', async function() {
-        await session.restore();
+      test('is not authenticated', async function(assert) {
+        try {
+          await session.restore();
+        } catch (e) {
+          // ignore
+        }
```

see above – not sure why we're hiding the error?